src/kOS/Binding/FlightControlManager.cs

```diff
@@ -201,6 +201,8 @@ public void SelectAutopilotMode(object autopilotMode)
             if (autopilotMode is Direction)
             {
                 //TODO: implment use of direction subclasses.
+                throw new Safe.Exceptions.KOSException(
```

Comment:
Yes, I think there should be proper exceptions for wrong types on both these setters. I'll put t there.
But this particular branch looks interesting, as if somebody wanted to add the option to feed the Direction directly into SAS... 
Anyway, accidentally feeding `prograde` instead of `"prograde"` could be a way too likely mistake not to put a special message for trying to feed a direction.